docs/DataProviders.md

```diff
@@ -66,6 +66,7 @@ The react-admin project includes 4 Data Providers:
 Developers from the react-admin community have open-sourced Data Providers for many more backends:
 
 * **[Django Rest Framework](https://www.django-rest-framework.org/)**: [synaptic-cl/ra-data-drf](https://github.com/synaptic-cl/ra-data-drf)
+* **[Express & Mongoose](https://www.npmjs.com/package/ra-data-simple-rest-expressjs-mongoose)**: [ra-data-simple-rest-expressjs-mongoose]
```

Comment:
If I read correctly, this is not a dataProvider, but a server-side module to fit with the provider, am I right? If so, it shouldn't appear here, but on the "miscellaneous" section of the Ecosystem doc. 

also, ti publish the link, you must specify a License in your library